buildscripts/manylinux2014/README.md

```diff
@@ -31,6 +32,7 @@ Available Python installations (`<pyver>`) are:
 - cp36-cp36m
 - cp37-cp37m
 - cp38-cp38
+- cp39-cp39
 
```

Comment:
- cp310-cp310

There is Python 3.10 in manylinux2014 as well (beta).

Reply:
Thank you for the info, I don't think we will want to build Python 3.10 for the time being. Most likely for the next release.